src/test/java/ay2122s1_cs2103t_w16_2/btbb/model/booking/UniqueBookingListTest.java

```diff
@@ -0,0 +1,94 @@
+package ay2122s1_cs2103t_w16_2.btbb.model.booking;
+
+import static ay2122s1_cs2103t_w16_2.btbb.testutil.Assert.assertThrows;
+import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalBookings.BOOKING_FOR_ALICE;
+import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalBookings.BOOKING_FOR_BENSON;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import ay2122s1_cs2103t_w16_2.btbb.exception.NotFoundException;
+
+class UniqueBookingListTest {
```

Comment:
I added because eventually we need to implement delete and edit. But on hindsight, i think i should not have done it since the person doing it should do the tests. Should I remove it?

I think i missed the `setBookings`. I will add it.